Cython/Compiler/Symtab.py

```diff
@@ -2237,7 +2237,8 @@ def declare_cfunction(self, name, type, pos,
                         # TODO(robertwb): Make this an error.
                         warning(pos,
                             "Compatible but non-identical C method '%s' not redeclared "
-                            "in definition part of extension type '%s'.  This may cause incorrect vtables to be generated." % (name, self.class_name), 2)
+                            "in definition part of extension type '%s'.  "
+                            "This may cause incorrect vtables to be generated." % (name, self.class_name), 2)
```

Comment:
Similarly, here, moving the formatting arguments to the next line makes it easier to see what is being used.

Reply:
I humbly suggest that you put rules which are not easily detectable by `pycodestyle` (or other packages) in a developer style guide somewhere (in the `contributing.md` maybe ?). 

It would save you some time during code review and it would save contributors some style rectification commits.

It is possible for contributors to notice that this style pattern is common in the codebase, but it is not possible for them to know that this is a rule which you apply during code review.

Reply:
You obviously couldn't know my preferences. But that's the thing, it's my preference, and it's not currently well visible in (the old parts of) the code base. And since this PR is all about changing and enforcing style, I had to tell you that that's not how I want the style to be.

Not sure if coala could help here, but style issues like this should best be solved automatically and not require manual interaction at all.